src/aligners/tests.rs

```diff
@@ -383,4 +383,183 @@ mod dt {
             }],
         ));
     }
+    //HomoPolymer tests
+
+    #[test]
+    fn ins_homopolymer_cost() {
+        test(AffineCost::new(
+            Some(1),
+            Some(1),
```

Comment:
I think this gives linear insertion of cost 1, and homopolymer insertion of cost 2+2*len. But deletions are now allowed.

Best to change the args to `sub: Some(1), ins: None, del: Some(1), affine: [... as it is now ..]`